common-lib/src/main/scala/com/gu/mediaservice/lib/aws/DynamoDB.scala

```diff
@@ -238,10 +238,10 @@ class DynamoDB(config: CommonConfig, tableName: String) {
     case value => value
   }
 
-  def valueMapWithNullForEmptyString(value: Map[String, String]) = {
+  def valueMapWithNullForEmptyString(value: Map[String, JsValue]) = {
     val valueMap = new ValueMap()
-    value.map     { case(k, v) => (k, if (v == "") null else v) }
-         .foreach { case(k, v) => valueMap.withString(k, v) }
+    value.map     { case(k, v) => (k, if (v == JsNull) null else v) }
+         .foreach { case(k, v) => valueMap.withJSON(k, Json.stringify(v)) }
```

Comment:
Would be good to have some tests against this change.

Reply:
@akash1810 I think this is done now?